content/en/docs/releases/supported-releases/index.md

```diff
@@ -70,9 +70,9 @@ Please keep up-to-date and use a supported version.
 
 | Minor Releases   | Patched versions with no known CVEs                  |
 | ---------------- | ---------------------------------------------------- |
-| 1.19.x           | 1.19.0+                                              |
-| 1.18.x           | 1.18.2+                                              |
-| 1.17.x           | 1.17.5+                                              |
+| 1.19.x           | 1.19.2+                                              |
+| 1.18.x           | 1.18.4+                                              |
```

Comment:
We ended up having to use 1.18.5 because the charts had already been published for 1.18.4

Reply:
I forgot to update that table, I've put in the correct versions (1.17.8, 1.18.5, 1.19.3)